site/content/blog/2020-12-01-whats-new-in-svelte-december-2020.md

```diff
@@ -0,0 +1,65 @@
+---
+title: What's new in Svelte: December 2020
+description: TBD
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+It's the last "What's new in Svelte" of the year... To round out the season, this month's coverage includes _things_!
```

Comment:
What is meant by _things_? I would prefer a more specific wording, like "a lot of showcases".

Reply:
Sorry this is a placeholder for when it's closer to the end of the month. The "TBD" in the description is also a placeholder. :)